spec/services/messages/send_push_spec.rb

```diff
@@ -0,0 +1,19 @@
+require "rails_helper"
```

Comment:
I suggest adding a test for the case when the push is not needed.

Reply:
I added a new test.